tests/Makefile.am

```diff
@@ -0,0 +1,89 @@
+# Copyright 2013-present Barefoot Networks, Inc.
```

Comment:
added an example in backend/p4test/tests to show how to extend unit test with custom backend.